language/de-DE/mod_stats.ini

```
@@ -11,7 +11,7 @@ MOD_STATS_CACHING="Zwischenspeicherung"
 MOD_STATS_FIELD_COUNTER_LABEL="Zugriffszähler"
 MOD_STATS_FIELD_INCREASECOUNTER_LABEL="Zähler hochsetzen"
```

```suggestion
MOD_STATS_FIELD_INCREASECOUNTER_LABEL="Zähler erhöhen"
```

siehe: https://github.com/joomlagerman/joomla/blob/1d4ef9964d08708be51c7547e7ef5f4510b20039/administrator/language/de-DE/mod_stats_admin.ini#L12